php/WP_CLI/CommandWithTranslation.php

```diff
@@ -125,6 +143,106 @@ public function install( $args, $assoc_args ) {
 	}
 
 	/**
+	 * Updates the active translation of core, plugins, and themes.
+	 *
+	 * [--dry-run]
+	 * : Preview which translations would be updated.
+	 *
+	 * @subcommand update
+	 */
+	public function update( $args, $assoc_args ) {
+
+		// Ignore updates for the default locale.
+		if ( 'en_US' == get_locale() ) {
+			\WP_CLI::success( "Translations updates are not needed for the 'English (US)' locale." );
+
+			return;
+		}
+
+		wp_clean_update_cache(); // Clear existing update caches.
+		wp_version_check();  // Check for Core updates.
+		wp_update_themes();  // Check for Theme updates.
+		wp_update_plugins(); // Check for Plugin updates.
```

Comment:
> I'm not sure about this, because these functions do not check only for translation updates.

The comment is for why the function calls are in this location though. And this location is specific to translations.